### ISSUE TITLE
link to chrome doesn't work

### ISSUE THREAD

**Issue by Kwaskoff** [Ragnarokkr/youtube-audio-library-downloader#1]
Check it, pls

**Comment by kraftydevil** [comment]
same issue here, https://chrome.google.com/webstore/detail/youtube-audio-library-dow/ajpanpjphikapgmodifpmcomgicdgpnf is broken. Is this project dead? If not, can you relist it on the extension store? Thanks

**Comment by ghost** [comment]
yes, this is the only application that I found online to download all youtube audio libraries according to filters but it's dead.
